### PR TITLE
More GraphQL operations

### PR DESCRIPTION
**Pull request by nikhilunni**
## Change description

Excruciatingly painful to work with GSON, but here it is

## Type of change
- [ ] Bug fix (fixes an issue)
- [ ] New feature (adds functionality)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] Application changes have been tested thoroughly
- [ ] Automated tests covering modified code pass

### Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### Code review 

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [ ] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

### CODE REVIEW

File: src/main/java/com/opsmatters/newrelic/api/httpclient/GsonMessageBodyHandler.java [hunk 1]
```diff
@@ -40,8 +40,8 @@
 import com.google.gson.GsonBuilder;
 import com.google.gson.JsonSyntaxException;
 import com.google.gson.reflect.TypeToken;
-import com.opsmatters.newrelic.api.httpclient.deserializers.graphql.GraphQLResponseDeserializer;
-import com.opsmatters.newrelic.api.httpclient.serializers.graphql.GraphQLResponseSerializer;
+import com.opsmatters.newrelic.api.httpclient.deserializers.graphql.*;
```
Comment (FoodProduct) [inline src/main/java/com/opsmatters/newrelic/api/httpclient/GsonMessageBodyHandler.java]: Is this change intentional? Do we need both this line and the line below?

File: src/main/java/com/opsmatters/newrelic/api/model/graphql/NrqlSuccessResponse.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
+package com.opsmatters.newrelic.api.model.graphql;
+
+import com.google.gson.JsonArray;
+
+import java.util.List;
+
+/**
+ * Represents Graph QL response data.
```
Comment (FoodProduct) [inline src/main/java/com/opsmatters/newrelic/api/model/graphql/NrqlSuccessResponse.java]: nit: I've noticed this a few times in this repo, but it's typically stylized as one word
```suggestion
 * Represents GraphQL response data.
```

**Review by FoodProduct — APPROVED** [review]
GSON seems pretty not DRY 😅

**Review by FoodProduct — APPROVED** [review]

**Review by att14 — APPROVED** [review]
